src/main/resources/liquibase/crop_changelog/19_4_0.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v19.4.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cvterm" columnName="is_system"/>
+			</not>
+		</preConditions>
+		<comment>Adding is_system column</comment>
+        <sql>
+			ALTER TABLE cvterm ADD COLUMN is_system INT(11) NOT NULL DEFAULT '0' AFTER is_relationshiptype;
+        </sql>
+    </changeSet>
+
+	<changeSet author="cuenyad" id="v19.4.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">SELECT count(1) > 0 FROM cvterm where is_system = 0 and name in ('TRIAL_INSTANCE','LOCATION_ID','LOCATION_NAME','EXPT_DESIGN','FIELDMAP COLUMN'
+				,'FIELDMAP RANGE','OBS_UNIT_ID','COL','ROW','BLOCK_NO','PLOT_NO','REP_NO','ENTRY_CODE'
+			    ,'ENTRY_NO','DESIGNATION','GID','MGID','FGID','GROUPGID','EXPT_DESIGN_SOURCE','DATASET_NAME'
+			    ,'DATASET_TITLE','LOCATION_NAME','DESIG','CROSS','ENTRY_TYPE', 'PLOT_CODE', 'CHECK_INTERVAL'
+			    ,'CHECK_PLAN','CHECK_START','PLOT_NUMBER_AP_text','INSTANCE_NUMBER_AP_text','REP_NUMBER_AP_text'
+			    ,'PLANT_NUMBER_AP_text','PLOTCODE_AP_text');
+			</sqlCheck>
+		</preConditions>
+		<comment>Mark system variables</comment>
+        <sql>
+			update cvterm cv INNER JOIN cvterm cv1 ON cv.cvterm_id = cv1.cvterm_id  set cv.is_system = 1
+			where cv1.name in ('TRIAL_INSTANCE','LOCATION_ID','LOCATION_NAME','EXPT_DESIGN','FIELDMAP COLUMN'
+				,'FIELDMAP RANGE','OBS_UNIT_ID','COL','ROW','BLOCK_NO','PLOT_NO','REP_NO','ENTRY_CODE'
+			    ,'ENTRY_NO','DESIGNATION','GID','MGID','FGID','GROUPGID','EXPT_DESIGN_SOURCE','DATASET_NAME'
+			    ,'DATASET_TITLE','LOCATION_NAME','DESIG','CROSS','ENTRY_TYPE', 'PLOT_CODE', 'CHECK_INTERVAL'
```

Comment:
For the ones that are used in TermId, I suggest to identify them by cvterm_id instead than name. 

The uniqueness of the cvterm table is not given by the name, so if there is more than one with the same name and different type, they will also be marked as system variables. Add cv_id = 1040 and is_obsolete = 0.